Lib/urllib/parse.py

```diff
@@ -417,9 +425,18 @@ def _checknetloc(netloc):
 def urlsplit(url, scheme='', allow_fragments=True):
     """Parse a URL into 5 components:
     <scheme>://<netloc>/<path>?<query>#<fragment>
-    Return a 5-tuple: (scheme, netloc, path, query, fragment).
-    Note that we don't break the components up in smaller bits
-    (e.g. netloc is a single string) and we don't expand % escapes."""
+
+    The result is a SplitResult object, which is a named 5-tuple with
```

Comment:
This will actually be a `SplitResult` or a `SplitResultBytes` object, depending on the type of the `url` parameter.